api/src/pcapi/connectors/ems.py

```diff
@@ -81,6 +81,8 @@ def do_request(self, endpoint: str, payload: dict) -> Response:
         Perform the actual request, using mandatory headers
         and hashed payload.
         """
+        sentry_sdk.set_tag("ems_http_call", self.__class__.__name__)
```

Comment:
```suggestion
        sentry_sdk.set_tag("ems_http_call", endpoint)
```
On pourrait ainsi obtenir une info plus précise sur cet appel externe.

Reply:
bien vu